third_party/ibis/ibis_teradata/client.py

```diff
@@ -66,7 +66,7 @@ class TeradataClient(SQLClient):
     table_class = TeradataTable
     dialect = compiler.TeradataDialect
 
-    def __init__(self, host, user_name, password, port=1025, logmech='TD2', use_no_lock_tables=False):
+    def __init__(self, host, user_name, password, port=1025, logmech='TD2', use_no_lock_tables=False, convert_to_utc=True):
```

Comment:
We could consider making this flag `default_time_zone="GMT"` and using that in the SQL below?  Then if someone actually uses a non-utc timezone they could specify that as well